### PR TITLE
Add owncloud code style

### LINKED ISSUE #242: php-cs-fixer
@DeepDiver1975 
Any plans to implement ``php-cs-fixer`` as we have in core ?

### PR DESCRIPTION
**Pull request by patrickjahns**
# Motivation

closes #242

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,8 +1,14 @@
 {
+    "config": {
+        "platform": {
+            "php": "5.6.29"
```
Comment (patrickjahns) [inline composer.json]: 5.6 should be enough / better - will doublecheck

Comment (phil-davis) [reply composer.json]: Yes, we are up to about ``5.6.33`` now, and counting for another 6 months.

**Comment by mmattel** [comment]
Any update?

**Comment by patrickjahns** [comment]
@tomneedham - would like to sync this change with your work for the next release - do you see that this introduces any critical change?

**Comment by phil-davis** [comment]
Rebased. Let's see CI do its thing.

**Pull request merged**